frontend/assets/lang/en.json

```diff
@@ -6,14 +6,15 @@
     "CompanyBrowser":{
         "CompaniesList": "Companies list",
         "FoundCompanies": "Found companies",
-        "Search": "Search",
-        "Filter": "Filters"
-    },
-    "Comboboxes":{
-        "ComboboxCity": "Choose city",
-        "ComboboxField": "Choose field of study",
-        "ComboboxDepartment": "Choose department",
-        "ComboboxSpecialization": "Choose specialization"
+        "Search": "Search"
+    },
+    "TreeSelects":{
+        "TreeSelectCity": "Choose city",
+        "TreeSelectSpecialization": "Choose specialization"
+    },
+    "pagination":{
+        "next": "Next",
```

Comment:
Why this is lowercase, if every other key is capitalized?

Reply:
It's some default error key from Pagination component and it didn't seem to work with upper case.

I didn't research how to change it yet.